requirements.txt

```diff
@@ -17,15 +17,14 @@ llvmlite==0.36.0
 matplotlib==3.3.4
 nlpaug==1.1.3
 numba==0.53.1
-numpy==1.20.2
+numpy>=1.19.5
```

Comment:
You shouldn't have these changes. Please rebase onto the newest version of the `main` branch.

Reply:
Okay. I'll make it now.